packages/x-date-pickers-pro/src/DateRangeCalendar/DateRangeCalendar.tsx

```diff
@@ -173,8 +174,9 @@ const DateRangeCalendar = React.forwardRef(function DateRangeCalendar<TDate>(
     reduceAnimations,
     onMonthChange,
     defaultCalendarMonth,
-    rangePosition: rangePositionProps,
-    onRangePositionChange,
+    rangePosition: rangePositionProp,
```

Comment:
Nitpick: We sometimes use prefix such aliased props with `inner` and other times suffix with `Prop`.
Is there a preferred way to use? 🤔 
Or maybe we could land on one?

Reply:
I agree that we should unify at some point.
I'm not a fan of `inner`, depending on how you use your variable, calling the external value `inner` can be weird and I could expect `inner` to be the one defined inside.

I like the `Prop` suffix because it always clearly refer to where to data come from.

Reply:
Indeed, I second this opinion. Using `inner` prefix is somewhat misleading. `Prop` suffix does sound more correct, but at the same time, a bit more tedious to target with refex if need be.
Here are a few other feasible options:
- providedRangePosition
- passedRangePosition
- inRangePosition (this one is already used quite often)

Reply:
I went for the`in` prefix which is shorter and carries the exact same info as `Prop`